src/datachain/catalog/catalog.py

```diff
@@ -921,7 +920,7 @@ def enlist_sources_grouped(
         from datachain.query import DatasetQuery
 
         def _row_to_node(d: dict[str, Any]) -> Node:
-            del d["source"]
+            del d["file__source"]
```

Comment:
Tweaking just for tests to pass. This will all be refactored in followup PRs that deal with CLI and `Catalog.index()`